lineman/app/components/memberships_page/pending_invitations_card/pending_invitations_card.haml

```diff
@@ -0,0 +1,18 @@
+.blank
+  %section.pending-invitations-card{ng-if: 'canSeeInvitations && group.hasPendingInvitations()'}
+    %h3.lmo-card-heading{translate: 'pending_invitations_card.heading'}
+    %table.pending-invitations-card__pending-invitations.table
+      %thead
+        %tr
+          %th{translate: 'pending_invitations_card.email'}
+          %th{translate: 'pending_invitations_card.invitation_link'}
+          %th{translate: 'pending_invitations_card.cancel'}
+      %tbody
+        %tr{ng-repeat: 'invitation in group.pendingInvitations() track by invitation.id'}
+          %td.pending-invitations-card__recipient-email
+            {{invitation.recipientEmail}}
+          %td.pending-invitations-card__invitation-url
+            https://www.loomio.org/invitations/{{invitation.token}}
```

Comment:
I thought the point was to copy-paste and send to people?

In any case I want a card in here for adding a 'resend invitation' link to this card, which will make it much better.